File: kb-core-ui/harness/tests/fixtures/mcp-tools/repo/pkg/mathutil.go
package mathutil

// DefaultStart is the calculator's starting total.
const DefaultStart = 0

// Add returns the sum of two integers.
func Add(a, b int) int {
	return a + b
}

// Calculator accumulates a running total.
type Calculator struct {
	Total int
	Label string
}

// AddTo adds n to the calculator's total and returns the new total.
func (c *Calculator) AddTo(n int) int {
	c.Total = Add(c.Total, n)
	return c.Total
}

// Reset zeroes the calculator.
func (c *Calculator) Reset() {
	c.Total = DefaultStart
}

File: kb-core-ui/harness/tests/fixtures/python-basics/repo/mathutil.py
"""Simple math utilities."""

DEFAULT_START = 0


def add(a, b):
    """Return the sum of two numbers."""
    return a + b


class Calculator:
    """Accumulates a running total."""

    def __init__(self, total=DEFAULT_START):
        self.total = total

    def add_to(self, n):
        """Add n to the total and return the new total."""
        self.total = add(self.total, n)
        return self.total
